securetea/core.py

```diff
@@ -41,11 +42,26 @@ def __init__(self):
         if(args.twitter_api_key and args.twitter_api_secret_key and args.twitter_access_token and
```

Comment:
@mishal23 I don't think it will be good to enforce Twitter args for Telegram. Suppose, user wants to configure Telegram only, not Twitter, this `if` statement will ignore that, and the program will break.
Better approach will be a functional check for Telegram outside this `if` statement for twitter args. Otherwise this PR looks great! :+1: 
@rejahrehim What are your views on this?

Reply:
Yes, that's a valid point. Let's see what @rejahrehim and other developers have to say on it.

Reply:
Yup, There should be no block in any case. We have to allow a user to configure his preferences. It would be better to implement a separate function to check the configurations and implement the application flow accordingly.  That is what I mentioned in the previous comment.
@mishal23 @abhisharma404  as you both are working on notification integrations any one of you can put a PR with these changes.

Reply:
@mishal23 We may need some changes in the previous code too.

Reply:
Updated.